roles/kubernetes_setup/tasks/main.yml

```diff
@@ -33,6 +33,39 @@
     - "{{ kube_controller_manager }}"
     - "{{ kube_scheduler }}"
 
+- name: Define kube 1.7 master nodes images
+  when: ansible_distribution == "Fedora" and ansible_distribution_major_version > '26'
```

Comment:
nit:  I prefer the list format for logical ands:

```
when:
  - ansible_distribution == "Fedora"
  - ansible_distribution_major_version > '26'
```

Reply:
For version comparisons, it is often better to use the `version_compare` filter, but in this case it should be fine as is.